src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,50 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LENGTH_OF_PASSWORD = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (isLoginIsNull(user)) {
+            throw new NullPointerException("Login can't be null");
+        }
+        if (isNotCorrectPassword(user)) {
```

Comment:
```suggestion
        if (!isCorrectPassword(user)) {
```
better use negation ! rather than "not" in method names. It is just confusing